release_notes.md

```diff
@@ -3,4 +3,5 @@
 - My change description (#PR)
  -->
 
+- Upgrade WebJobs to 3.0.20 ([Release Notes](https://github.com/Azure/azure-webjobs-sdk/releases/tag/v3.0.20))
```

Comment:
Can we summarize the 3.0.20 release here so it's easier to digest? In this case it's only one PR that's relevant: https://github.com/Azure/azure-webjobs-sdk/pull/2544. If we can put a sub-bullet maybe with the title of that PR?